stage1/init/init.go

```diff
@@ -228,6 +230,7 @@ func getArgsEnv(p *Pod, flavor string, systemdStage1Version string, debug bool)
 
 	switch flavor {
 	case "coreos":
+		args = append(args, filepath.Join(common.Stage1RootfsPath(p.Root), interpBin))
```

Comment:
Why do we need it? Is `LD_LIBRARY_PATH` not going to work if nspawnBin is executed directly?